sdk/Lusid.Sdk.Tests/tutorials/Instruments/ExoticExamples.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class ExoticExamples: DemoInstrumentBase
+    {
+        internal override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
```

Comment:
Consider adding comment describing why this is empty

Reply:
I think I'll add comment here when I add the valuation step (separate merge - hopefully by the end of this week). Not 100% sure it'll stay empty currently.